src/favorites/FavoriteCategoryButton.js

```diff
@@ -1,12 +1,15 @@
 import React from 'react';
 import Icon from '../widgets/Icon';
 
-const FavoriteCategoryButton = ({ name, onClick, isFavorited }) => {
+const FavoriteCategoryButton = ({ name, onClick, isFavorited, noTitle }) => {
   return (
-    <h2 className="mt-1 mb-0 text-xs-center">
```

Comment:
I think the matter is in category page we want to be able to favorite categories even by clicking on the category title (and not star only) By this change we will lose that possibility.